metpy/calc/kinematics.py

```diff
@@ -595,6 +595,10 @@ def storm_relative_helicity(u, v, heights, depth, bottom=0 * units.m,
             (positive_srh + negative_srh).to('meter ** 2 / second ** 2'))
 
 
+@deprecated('0.8', addendum=' This function has been replaced by the signed delta distance'
+                            'calculation lat_lon_grid_deltas and will be removed in MetPy'
+                            ' 0.11.',
+            pending=False)
 @exporter.export
 def lat_lon_grid_spacing(longitude, latitude, **kwargs):
```

Comment:
You know, we *could* make the duplicated code gods happy by making `lat_lon_grid_spacing` do:
```python
dx, dy = lat_lon_grid_deltas(...)
return np.abs(dx), np.abs(dy)
```